src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

```diff
@@ -40,6 +40,12 @@ class AppPipelineDsl extends CommonPipelineDsl implements Serializable {
     config.crossBrowserTest = true
   }
 
+  void enableParallelCrossBrowserTest(List<String> browsers = ['chrome', 'firefox', 'safari', 'microsoft'], int timeout = 120) {
```

Comment:
A general suggestion, why not have it as an overloaded method with additional parameter like parallelBrowsers and change code to behave based on this ? That might remove some duplicate lines.

Reply:
that's a really interesting idea @adusumillipraveen , I'll look into that for the actual `parallelCrossBrowserTest()` method which could definitely be renamed to overload the existing `crossBrowserTest()` method. 

But I think we'd still want a separate `enableParallelCrossBrowserTest()` here because I'm hoping most services will end up using the default parameters like 
```
  enableParallelCrossBrowserTest()
```
 so unless I'm misunderstanding we'd have to supply something to differentiate between the similar named methods like a 'parallel: true' value or something, which I'm not sure gives us much benefit over a different method name

Reply:
done